src/mdacli/__init__.py

```diff
@@ -1,3 +1,17 @@
+#!/usr/bin/env python3
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+#
+# Copyright (c) 2021 Authors and contributors
+#
+# Released under the GNU Public Licence, v2 or any higher version
+# SPDX-License-Identifier: GPL-2.0-or-later
 """Command line interfaces for MDAnalysis analysis classes."""
 
+from .cli import maincli, setup_clients
```

Comment:
The way we have the package set up you can avoid using relative imports. I would vote for absolute import that holds on an installation.

```python
from mdacli.cli import maincli, setup_clients
```

yet, see my next comment.

Reply:
Yes. good idea